execute-generators/build.gradle.kts

```diff
@@ -37,13 +37,15 @@ version = if (project.hasProperty("forceCI") || project.hasProperty("teamcity"))
 }
 
 
-val mpsConfiguration = configurations.create("mps")
-
 dependencies {
     implementation(kotlin("stdlib-jdk8", version = kotlinVersion))
     implementation("com.xenomachina:kotlin-argparser:$kotlinArgParserVersion")
-    mpsConfiguration("com.jetbrains:mps:$mpsVersion")
-    compileOnly(mpsConfiguration.resolve().map { zipTree(it) }.first().matching { include("lib/*.jar") })
```

Comment:
Why don't we need to include the lib/*.jars any longer?

Reply:
Those Jars from the complete mps distro are replaced by smaller Maven publications based on the MPS distro, which drastically reduces the size of all dependencies needed to be downloaded for the plugin build.